__tests__/app.test.js

```diff
@@ -159,3 +160,48 @@ describe("/api/articles", () => {
   })
 })
 
+describe.only("/api/articles/:article_id/comments", () => {
```

Comment:
.only - any tests that skip or isolate need to be removed before submitted a PR

Reply:
Hi, just removed the .only and all tests are passing, sorry about that!